lesson_02/ht_template/job1/tests/test_dal/test_local_disk.py

```diff
@@ -2,14 +2,27 @@
 Tests dal.local_disk.py module
 # TODO: write tests
 """
-from unittest import TestCase, mock
-
+from unittest import TestCase, main
+import os
 from lesson_02.ht_template.job1.dal.local_disk import save_to_disk
 
 
+BASE_DIR = "D:\DE_HW\lesson_02\ht_template"
+DATE = "2022-08-10"
+RAW_DIR = os.path.join(BASE_DIR, "raw", "sales", DATE)
+SAMPLE_DATA = [{"client": "Michael Wilkerson", "purchase_date": "2022-08-10", "product": "Vacuum cleaner", "price": 346}]
+
+
 class SaveToDiskTestCase(TestCase):
     """
     Test dal.local_disk.save_to_disk function.
-    # TODO: implement
     """
-    pass
\ No newline at end of file
+    def test_save_to_disk_success(self):
+        save_to_disk(SAMPLE_DATA, RAW_DIR, DATE)
+        is_saved = os.path.exists(RAW_DIR)
+        self.assertTrue(is_saved)
```

Comment:
You check that the saving directory exists only, not the files you saved. 
The right way would be to create a temporary directory first, write the data and check if the data is saved properly